news.js

```diff
@@ -0,0 +1,10 @@
+(function (exports) { 
+    function News(headlines) {
```

Comment:
It doesn't look like we're currently using this `News` component in the `index.html` code. What is the responsibility of this module and how might it eventually fit into the page?